<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="22528" started-at="2019-12-15T15:25:06Z" finished-at="2019-12-15T15:25:28Z">
    <groups>
      <group name="cucumber">
        <method signature="TestRunner.feature(cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:MyRunner.TestRunner@887af79]" name="feature" class="MyRunner.TestRunner"/>
      </group> <!-- cucumber -->
    </groups>
    <test name="Default test" duration-ms="22528" started-at="2019-12-15T15:25:06Z" finished-at="2019-12-15T15:25:28Z">
      <class name="MyRunner.TestRunner">
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:MyRunner.TestRunner@887af79]" name="setUpClass" is-config="true" duration-ms="344" started-at="2019-12-15T20:55:06Z" finished-at="2019-12-15T20:55:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="feature(cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:MyRunner.TestRunner@887af79]" name="feature" duration-ms="21273" started-at="2019-12-15T20:55:06Z" description="Runs Cucumber Feature" data-provider="features" finished-at="2019-12-15T20:55:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[feature to login]]>
              </value>
            </param>
          </params>
          <exception class="cucumber.runtime.CucumberException">
            <message>
              <![CDATA[java.lang.AssertionError: expected [CRMPRO  - CRM software for customer relationship management, sales, and support.] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[cucumber.runtime.CucumberException: java.lang.AssertionError: expected [CRMPRO  - CRM software for customer relationship management, sales, and support.] but found []
	at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
	at MyRunner.TestRunner.feature(TestRunner.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: java.lang.AssertionError: expected [CRMPRO  - CRM software for customer relationship management, sales, and support.] but found []
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at stepDefinitions.GeneralSteps.validateMethod(GeneralSteps.java:38)
	at âœ½.Then user validates "title" as "CRMPRO  - CRM software for customer relationship management, sales, and support." in "home" page(login.feature:7)
]]>
            </full-stacktrace>
          </exception> <!-- cucumber.runtime.CucumberException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- feature -->
        <test-method status="PASS" signature="tearDownClass()[pri:0, instance:MyRunner.TestRunner@887af79]" name="tearDownClass" is-config="true" duration-ms="818" started-at="2019-12-15T20:55:28Z" finished-at="2019-12-15T20:55:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- MyRunner.TestRunner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
